### PR TITLE
add iOS/ipadOS 13, macOS 10.15, Safari 13

### LINKED ISSUE #2124: iPadOS 13
see serbanghita/Mobile-Detect#795 for more details.

### PR DESCRIPTION
**Pull request by mimmi20**
Fixes #2124.

### CODE REVIEW

File: tests/issues/issue-2124.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        'ua' => 'Mozilla/5.0 (iPad; CPU OS 13_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Mobile/15E148 Safari/604.1',
+        'properties' => [
+            'Comment' => 'Mobile Safari 13.0',
+            'Browser' => 'Safari',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Apple Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '13.0',
+            'Platform' => 'iOS',
```
Comment (jaydiablo) [inline tests/issues/issue-2124.php]: Why is this one `iOS` in Standard/Full but `ipadOS` on Lite?

File: resources/platforms.json [hunk 2]
```diff
@@ -1761,6 +1772,31 @@
       "Platform_Bits": 32
     }
   },
+  "ipadOS_13_0": {
+    "match": "*CPU OS 13_0* like Mac OS X*",
+    "lite": false,
+    "standard": true,
+    "inherits": "iOS",
```
Comment (jaydiablo) [inline resources/platforms.json]: Shouldn't this inherit `ipadOS`?

File: tests/issues/issue-2124.php (hunk 1 above)
Comment (mimmi20) [reply tests/issues/issue-2124.php]: fixed

File: resources/platforms.json (hunk 2 above)
Comment (mimmi20) [reply resources/platforms.json]: fixed

**Pull request merged**